addon/components/imgix-photo.js

```diff
@@ -78,6 +78,14 @@ export default class ImgixPhoto extends Component {
   }
 
   get imgSrc() {
+    if (this.args.url.match(/\?/)) {
+      if (this.args.params) {
+        console.warn('ImgixPhoto was given a `url` which contains query params. Ignoring `params` attribute.');
+      }
```

Comment:
we use `w` and `h` both for appending it to the URL and setting the width + height of the `img` tag of the component. We might still need the ability to set both width + height of the `img` tag, whether that is still done through params or not is TBD, but my idea was to drop params altogether to avoid confusion, maybe add new arguments (width and height) specifically? For now, this serves the purpose though with minimal impact on the consuming apps.